test/v3_helper.rb

```diff
@@ -8,3 +8,24 @@
 Approvals.configure do |c|
   c.approvals_path = './test/fixtures/approvals/'
 end
+
+module Minitest
+  module Assertions
+    def assert_files_in_zip(files, zip_file)
```

Comment:
I would just put this directly in the test that uses them, since only that one file needs it.